src/services/SocketService/socketRequestService.js

```diff
@@ -161,11 +165,34 @@ const declineOffer = ({ chatRoomId, offerId, userType, socket }) => {
   );
 };
 
+const archiveChatRoom = ({ chatRoomId, socket }) => {
+  socket.emit(
+    EMIT_ARCHIVE_CHATROOM,
```

Comment:
I really do not think you should use sockets for archiving or unarchiving a chatroom. Shouldn't it be a POST request?? Imagine Buyer A archives a chatroom. Seller B may not want it to be archived. But it will still disappear from Seller B's chat list because of this socket thing. It is extremely weird.